README.md

```diff
@@ -1 +1,11 @@
+# ROS2 GitHub Action CI Template
 
+A template repo to help ROS2 developers create a GitHub repository with the ROS2 CI GitHub Action.
```

Comment:
Avoid passive voice, compare with https://github.com/actions/javascript-action

"Use this template to bootstrap the creation of a JavaScript action.🚀"